admin/components/Settings/index.js

```diff
@@ -0,0 +1,302 @@
+  siteIdStatus,
+  ssrServerStatus,
+  staticServerStatus,
+  ampServerStatus,
+}) => (
+  <Box margin={{ horizontal: "auto", top: "40px" }} width="608px">
+    <Notification align="center" margin={{ bottom: "20px" }}>
+      <StyledParagraph margin={{ vertical: "0" }}>
+        <strong>{notification.highlight} </strong>
+        {notification.content}
+      </StyledParagraph>
+    </Notification>
+    <Options margin={{ bottom: "24px" }}>
+      <Header margin={{ horizontal: "0", vertical: "0" }}>
+        {formTitleText}
+      </Header>
+      <Form>
+        <FormField label={fieldSiteId.label}>
+          <StyledTextInput
+            status={siteIdStatus}
+            placeholder={fieldSiteId.placeholder}
+            value={siteId}
+            onChange={setSiteId}
+          />
+        </FormField>
+        <FormField label={fieldSsrServer.label}>
+          <StyledTextInput
+            status={ssrServerStatus}
+            placeholder={fieldSsrServer.placeholder}
+            value={ssrServer}
+            onChange={setSsrServer}
+          />
+        </FormField>
+        <FormField label={fieldStaticServer.label}>
+          <StyledTextInput
+            status={staticServerStatus}
+            placeholder={fieldStaticServer.placeholder}
+            value={staticServer}
+            onChange={setStaticServer}
+          />
+        </FormField>
+        <FormField label={fieldAmpServer.label}>
+          <StyledTextInput
+            status={ampServerStatus}
+            placeholder={fieldAmpServer.placeholder}
+            value={ampServer}
+            onChange={setAmpServer}
+          />
+        </FormField>
+        <Box
+          direction="row"
+          justify="between"
+          align="start"
+          margin={{ bottom: "18px" }}
+        >
+          <Box direction="row" justify="between" align="center" width="262px">
+            <Paragraph margin={{ vertical: "0", left: "12px", right: "20px" }}>
+              {fieldForceFrontpage.label}
+            </Paragraph>
+            <CheckBox
+              toggle
+              checked={frontpageForced}
+              onChange={setFrontpageForced}
+            />
+          </Box>
+          <Comment margin={{ vertical: "0" }}>
+            {fieldForceFrontpage.comment}
+          </Comment>
+        </Box>
+        <Box
+          direction="row"
+          justify="between"
+          align="center"
+          margin={{ bottom: "18px" }}
+        >
+          <Box direction="row" justify="between" align="center" width="262px">
+            <Paragraph margin={{ vertical: "0", left: "12px", right: "20px" }}>
+              {fieldHtmlPurifier.label}
+            </Paragraph>
+            <CheckBox
+              toggle
+              checked={htmlPurifierActive}
+              onChange={setHtmlPurifierActive}
+            />
+          </Box>
+          <Button
+            label={fieldHtmlPurifier.button}
+            onClick={purgeHtmlPurifierCache}
```

Comment:
Maybe we can use a dialog here, like this one. It's very simple to use:
https://storybook.grommet.io/?selectedKind=Layer&selectedStory=Center&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel